### PR TITLE
Fix2 electricity price prediction.

### PR DESCRIPTION
**Pull request by b0661**
Normalize electricity price prediction to €/Wh.
Provide electricity price prediction by €/kWh for convenience.

Allow to configure electricity price charges by €/kWh.

Also added error page to fastapi rest server to get rid of annoying unrelated fault messages during testing.

### CODE REVIEW

File: docs/akkudoktoreos/openapi.json [hunk 1]
```diff
@@ -2523,7 +2523,7 @@
                 "type": "null"
               }
             ],
-            "title": "Elecprice Charges",
+            "title": "Elecprice Charges Kwh",
```
Comment (NormannK) [inline docs/akkudoktoreos/openapi.json:2526]: kWh

File: docs/akkudoktoreos/openapi.json [hunk 2]
```diff
@@ -5163,7 +5163,7 @@
...
```
Comment (NormannK) [inline docs/akkudoktoreos/openapi.json:5166]: kWh

**Review by NormannK — APPROVED** [review]

**Comment by Lasall** [comment]
Let's update the default value (config) and also in docker-compose (so users can see directly what values they probably have to adjust).

File: docs/akkudoktoreos/openapi.json (hunk 1 above)
Comment (b0661) [reply docs/akkudoktoreos/openapi.json:2526]: Das ist autogeneriert aus `elecprice_charges_kwh`. Keine Ahnung wie man openapi hier helfen kann.

File: docs/akkudoktoreos/openapi.json (hunk 2 above)
Comment (b0661) [reply docs/akkudoktoreos/openapi.json:5166]: Das ist autogeneriert aus `elecprice_charges_kwh`. Keine Ahnung wie man openapi hier helfen kann.

**Comment by b0661** [comment]
> Let's update the default value (config) and also in docker-compose (so users can see directly what values they probably have to adjust).

Default config and docker config updated. Additional made application server not to start in docker image as was a wish on signal.

**Pull request merged**